lxd/container.go

```diff
@@ -32,6 +33,24 @@ import (
 	"github.com/lxc/lxd/shared/units"
 )
 
+func init() {
+	// Expose containerLoadNodeAll to the device package converting the response to a slice of InstanceIdentifiers.
+	// This is because container types are defined in the main package and are not importable.
+	device.InstanceLoadNodeAll = func(s *state.State) ([]device.InstanceIdentifier, error) {
+		containers, err := containerLoadNodeAll(s)
+		if err != nil {
+			return nil, err
+		}
+
+		instances := []device.InstanceIdentifier{}
```

Comment:
Just a small nit, but maybe call that `identifiers` so that when we go ahead and rename most of our functions from `container` to `instance` we don't end up with a variable name conflict in here.